### DISCUSSION TITLE
Better way to make stages

### DISCUSSION THREAD

**Discussion by Lazelmaybe** [JordanSantiagoYT/FNF-JS-Engine#713] [Ideas]
adding a stage and or background is kind of difficult, i think you should make an easier way to make one!

**Comment by JordanSantiagoYT** [comment]
why not.. just use the pe 1.0 stage edito

**Reply by moxie-coder** [reply]
literally

**Comment by TheStinkern** [comment]
> why not.. just use the pe 1.0 stage edito

or look up a tutorial to make things easier

**Comment by imavie** [comment]
its my job now....
